rest_framework/renderers.py

```diff
@@ -42,9 +43,9 @@ class BaseRenderer:
     All renderers should extend this class, setting the `media_type`
     and `format` attributes, and override the `.render()` method.
     """
-    media_type = None
-    format = None
-    charset = 'utf-8'
+    media_type = None  # type: Optional[str]
```

Comment:
Is there not a `?` shortcut for `Optional` (like they have in Swift)? (Just wondering...)

Reply:
As far as I am aware, there isn't a shortcut for `Optional`.